packages/core/src/App/Containers/Modals/ready-to-verify-modal/ready-to-verify-modal.tsx

```diff
@@ -45,8 +43,8 @@ const ReadyToVerifyModal = observer(() => {
             has_close_icon={false}
             onEscapeButtonCancel={onClose}
         >
-            <Text align='center' size={is_mobile ? 'xxs' : 'xs'}>
-                {has_deposited_for_first_time
+            <Text align='center' size={isMobile() ? 'xxs' : 'xs'}>
```

Comment:
why we remove `is_mobile` with `isMobile()` ?

Reply:
i will change it back